sentry_sdk/integrations/asgi.py

```diff
@@ -178,11 +182,27 @@ async def _run_app(self, scope, callback):
                     with hub.start_transaction(
                         transaction, custom_sampling_context={"asgi_scope": scope}
                     ):
-                        # XXX: Would be cool to have correct span status, but we
-                        # would have to wrap send(). That is a bit hard to do with
-                        # the current abstraction over ASGI 2/3.
                         try:
-                            return await callback()
+
+                            async def _sentry_wrapped_send(event):
+                                # type: (Dict[str, Any]) -> Any
+                                is_http_response = (
+                                    event["type"] == "http.response.start"
```

Comment:
Is `event["type"]` guaranteed to always be there for any event? Would it make sense to `event.get("type")` instead?

Reply:
done. better save then sorry!